prime-angular-frontend/src/app/modules/site-registration/pages/remote-user-page/remote-user-page.component.ts

```diff
@@ -92,8 +96,13 @@ export class RemoteUserPageComponent extends AbstractEnrolmentPage implements On
   }
 
   public licenceFilterPredicate() {
-    return (licenceConfig: LicenseConfig) =>
-      this.enrolmentService.hasAllowedRemoteAccessLicences(licenceConfig);
+    if (this.site.remoteAccessTypeCode) {
+      return (licenceConfig: LicenseConfig) =>
+        licenceConfig.remoteAccessTypeLicenses.length &&
+        licenceConfig.remoteAccessTypeLicenses.some((r) => r.remoteAccessTypeCode === this.site.remoteAccessTypeCode);
+    } else {
+      return null;
```

Comment:
Is `return false;` more semantically correct?

Reply:
Updated.